Mathlib/RingTheory/Generators.lean

```diff
@@ -215,6 +215,7 @@ def baseChange {T} [CommRing T] [Algebra R T] (P : Generators R S) : Generators
 end Construction
 
 variable {R' S'} [CommRing R'] [CommRing S'] [Algebra R' S'] (P' : Generators R' S')
+variable [Algebra R R'] [Algebra S S']
```

Comment:
This one looks wrong (and the other one in the same file also): you should'nt need to add new assumptions without removing other ones.

Reply:
Fixed.